lib/vim-flavor/facade.rb

```diff
@@ -46,7 +46,7 @@ def test(files_or_dirs, options)
           %Q{ '#{runner}' \
               #{runtime_paths.flat_map {|p| ['-d', p]}.shelljoin} \
               #{files_or_dirs.shelljoin} }
-        succeeded = system(command)
+        succeeded = system("bash", "-c", command)
```

Comment:
Would you mind replacing `"` to `'`? `'` is used in existing code unless `"` is necessary.